tox.ini

```diff
@@ -12,12 +12,28 @@ setenv =
 deps =
 	-r{toxinidir}/requirements.txt
 	django111: Django<1.12
-	django20: Django<2.1
-	django21: Django<2.2
-	django-oauth-toolkit>=1.1.2,<1.2
-	djangorestframework>=3.7,<3.8
-	pyjwt>=1.6.4,<1.7
-	py27: mock
+	django111: django-oauth-toolkit>=1.1.2,<1.2
```

Comment:
This does not make too much sense to me if you need to define then `!django111`. Better as it was before.

Reply:
Basically because the latest dot-version doesn't support Django 1.11 anymore.
And if I don't explicitly set `!django111` you get something like this:
```ERROR: Double requirement given: django-oauth-toolkit<=1.12,>=1.1.2 (already in django-oauth-toolkit<1.2,>=1.1.2, name='django-oauth-toolkit')```

Reply:
Currently I'm not aware of any elegant solution to this, but I'm open to suggestions.